### PR TITLE
Improvement for Jira 1870 - provide ability to search experiments by jobId in the experiment statistics tab under admin dashboard

### PR DESCRIPTION
**Pull request by shivamriky**
I have added functionality in the PGA to search experiments using the jobId. It uses an existing API function searchExperiment() to first search the experiments using a new field 'JobId'. This new field is added to the experiment_model.thrift file.
After searching the corresponding experimentid using jobid, the normal flow of information follows i.e, the experiment Id is used to search a detailed view of the experiment and then presented to the user. 

The below screenshot displays the functionality. 

image

### CODE REVIEW

File: modules/ide-integration/src/main/resources/pga/airavata-php-gateway/app/controllers/ExperimentController.php [hunk 1]
```diff
@@ -129,8 +129,42 @@ public function createSubmit()
...
+            $limit = 1;
+            $pageNo = 0;
+            $inputs["search-key"] = "jobId";
+            $inputs["search-value"] = $_GET['jobId'];
+            try {
+                $expContainer = ExperimentUtilities::get_expsearch_results_with_pagination($inputs, $limit,
+                    $pageNo);
+            } catch (AuthorizationException $ae) {
+
+                Log::error("Experiment wasn't found", array("message" => $enf->getMessage(), "username" => Session::get("username"), "gateway_id" => Session::get("gateway_id")));
```
Comment (DImuthuUpe) [inline modules/ide-integration/src/main/resources/pga/airavata-php-gateway/app/controllers/ExperimentController.php]: Change the error message. This is an authorization error

File: modules/registry/registry-core/src/main/java/org/apache/airavata/registry/core/repositories/expcatalog/ExperimentSummaryRepository.java [hunk 2]
```diff
@@ -51,6 +56,17 @@
             throw new RegistryException("GatewayId is required");
         }
 
+        if (filters.get(DBConstants.Job.JOB_ID) != null) {
+            logger.debug("Filter Experiments by JobId");
+            queryParameters.put(DBConstants.Job.JOB_ID, filters.get(DBConstants.Job.JOB_ID));
+            String query_jobId = "SELECT P.experimentId FROM "
```
Comment (DImuthuUpe) [inline modules/registry/registry-core/src/main/java/org/apache/airavata/registry/core/repositories/expcatalog/ExperimentSummaryRepository.java]: Don't you need to filter by the gateway id as well?

Comment (shivamriky) [reply modules/registry/registry-core/src/main/java/org/apache/airavata/registry/core/repositories/expcatalog/ExperimentSummaryRepository.java]: As discussed, the Airavata API is filtering the experiments that are accessible by the user using the sharing service.

File: modules/ide-integration/src/main/resources/pga/airavata-php-gateway/app/controllers/ExperimentController.php (hunk 1 above)
Comment (shivamriky) [reply modules/ide-integration/src/main/resources/pga/airavata-php-gateway/app/controllers/ExperimentController.php]: I have fixed it.

**Pull request merged**